### DISCUSSION TITLE
Optical Flow (and other OpenCV tools)

### DISCUSSION THREAD

**Discussion by bits-by-brandon** [cables-gl/cables_docs#943] [Feature Requests]
There are a whole host of image processing tools that are openly available that would open a range of creative possibilities. I was looking at using Optical Flow to make some interactive particles. If something like OpenCV was available to use, it would give us so many new tools including:
- Optical Flow
- SLAM
- Depth processing
- Object Detection
- Video Stabilization

and many more!

There is an official way to build OpenCV for js with enscripten: https://docs.opencv.org/4.10.0/df/d0a/tutorial_js_intro.html

**Comment by steam0r** [comment]
the library seems to load just fine and using it with cables-ops like ImageElement and such should be easy. 
you could/should create a team around this and try implementing the ops you need there.

https://cables.gl/teams
